pandas-stubs/core/series.pyi

```diff
@@ -2076,24 +2076,41 @@ class Series(IndexOpsMixin[S1], NDFrame):
         numeric_only: _bool = ...,
         **kwargs,
     ) -> Scalar: ...
+    # Rename axis with `mapper`, `axis`, and `inplace=True`
```

Comment:
`frame.pyi` has four overloads: {`mapper` and `axis`, `index` and `columns`} × {`inplace=True`, `inplace=False`}. This reflects the [documentation](https://pandas.pydata.org/docs/reference/api/pandas.Series.rename_axis.html) suggestion to "Use either `mapper` and `axis` to specify the axis to target with `mapper`, or `index` and/or `columns`." So I have followed suit here, adding two overloads.